specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -3261,6 +3261,105 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/billingAccounts/default/lineOfCredit/default": {
+      "get": {
+        "tags": [
+          "LineOfCredits"
+        ],
+        "x-ms-examples": {
+          "LineOfCreditBySubscription": {
+            "$ref": "./examples/LineOfCreditBySubscription.json"
+          }
+        },
+        "operationId": "LineOfCredits_Get",
+        "description": "Get the current line of credit.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/LineOfCredit"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/billingAccounts/default/lineOfCredit/default/increase": {
+      "post": {
+        "tags": [
+          "LineOfCredits"
+        ],
+        "operationId": "LineOfCredits_Increase",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "IncreaseLineOfCreditBySubscription": {
+            "$ref": "./examples/IncreaseLineOfCreditBySubscription.json"
+          }
+        },
+        "description": "Increase the current line of credit.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IncreaseLineOfCreditRequestProperties"
+            },
+            "description": "Parameters supplied to the increase line of credit operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/LineOfCredit"
+            }
+          },
+          "202": {
+            "description": "Accepted. Line of credit increase is in progress.",
+            "headers": {
```

Comment:
instead of declaring headers, you should use:

        "x-ms-long-running-operation": true,
        "x-ms-long-running-operation-options": {
          "final-state-via": "azure-async-operation"
        }

Reply:
Sure, done.

Reply:
Added this:
"x-ms-long-running-operation": true,
        "x-ms-long-running-operation-options": {
          "final-state-via": "location"
        },

Reply:
@ 
As the api returns both location and azure-async operation header, I am wondering how this would translate well, as it specifies location only ?

"final-state-via": "location"

---
In reply to: [287507188](https://github.com/Azure/azure-rest-api-specs/pull/6054#discussion_r287507188) [](ancestors = 287507188)